### ISSUE TITLE
index file with a space in the name?

### ISSUE THREAD

**Issue by lavalamp** [Roman2K/scat#17]
It doesn't seem possible to pass a filename including a space to the index proc, or at least quoting it doesn't work. (I would never choose to name the file with a space but my script is using the name of the directory it's backing up, and naturally one of these has spaces in the name...)

**Comment by Roman2K** [comment]
@lavalamp That's right... As is, string arguments can't contain spaces, so paths can't contain spaces in`index` 🙈. No shame in having file/dir-names containing spaces, hehe! Definitely a case to handle, I'll take care of it when I'll have time. Or if you want to give it a try: [argparse/str.go](https://github.com/Roman2K/scat/blob/master/argparse/str.go) 😉
